### PR TITLE
Display license info

### PR DESCRIPTION
**Pull request by clanktron**
Replaces the signout button with a profile dropdown containing license information.

### CODE REVIEW

**Comment by clanktron** [comment]
depends on rancher-government-carbide/carbide-registry-api#32

File: assets/user.svg
Comment (dweomer) [inline assets/user.svg:1]: noun project or ... ?

File: components/navbar.vue [hunk 1]
```diff
@@ -4,19 +4,46 @@
...
+<style scoped>
+/* bootstrap's version of disabled greys them out */
+.custom-disabled {
+    pointer-events: none;
+}
+</style>
+
 <script setup lang="ts">
-const loggedIn = ref(false);
+    const loggedIn = ref(false);
```
Comment (dweomer) [inline components/navbar.vue:33]: this `const` stuff makes my head hurt. h8 ecmascript

**Review by dweomer — APPROVED** [review]

File: assets/user.svg
Comment (clanktron) [reply assets/user.svg:1]: It's from svgrepo.com

**Pull request merged**